composer.json

```diff
@@ -30,6 +30,7 @@
     },
     "require-dev": {
         "ext-pdo_sqlite": "*",
+        "composer/semver": "^1.7.4 || ^2 || ^3.2",
```

Comment:
Are there any reason why 3.1 would not work? :) 
I would propose to define it similar to other deps: `^1.7.4|^2.0|^3.0`. May it would be nice to add into `suggest` section too?

Reply:
You are right - I fixed the version constraints and added the package to the `suggest` section.